pygmt/tests/baseline/test_basemap_aliases.png.dvc

```diff
@@ -0,0 +1,4 @@
+outs:
+- md5: 9841c138346e468380aaef4b927d1fc3
+  size: 25016
+  path: test_basemap_aliases.png
```

Comment:
There isn't a `test_basemap_aliases` function, maybe a remainder from an old unused test?

Reply:
removed in 9b414a3